src/Components/ExtractLevelsMenu.jsx

```diff
@@ -41,6 +46,17 @@ export default function ExtractLevelsMenu({listOfOptions, icon, title}) {
     const normal2 = new Vector3(0, -1, 0)
     viewer.clipper.createFromNormalAndCoplanarPoint(normal1, modelCenter1)
     viewer.clipper.createFromNormalAndCoplanarPoint(normal2, modelCenter2)
+    console.log(h1)
```

Comment:
remove. please proofread the code, and delete all of the consoles.

Reply:
Addressed. Changed names.